admittance_controller/src/admittance_controller.cpp

```diff
@@ -587,13 +578,14 @@ controller_interface::return_type AdmittanceController::update()
   return controller_interface::return_type::OK;
 }
 
-void AdmittanceController::read_state_from_hardware(trajectory_msgs::msg::JointTrajectoryPoint & state)
+void AdmittanceController::read_state_from_hardware(
+  trajectory_msgs::msg::JointTrajectoryPoint & state)
 {
-  const auto joint_num = joint_names_.size();
-  auto assign_point_from_interface = [&, joint_num](
+  const auto num_joints = joint_names_.size();
+  auto assign_point_from_interface = [&, num_joints](
     std::vector<double> & trajectory_point_interface, const auto & joint_inteface)
```

Comment:
Spelling:  `inteface` --> `interface`

Not a fan of the auto here in `const auto & joint_interface`. It's hard to tell what the type is.

Reply:
I would keep auto because the type is very long and complicated. see [here](https://github.com/destogl/ros2_controllers/blob/41e5ae31d08e2790513ba8c2bd40b40867a3afb0/admittance_controller/include/admittance_controller/admittance_controller.hpp#L128)